src/stcal/alignment/util.py

```diff
@@ -157,28 +164,74 @@ def _get_axis_min_and_bounding_box(footprints: list[np.ndarray],
     ref_wcs : ~gwcs.wcs.WCS
         The reference WCS object.
 
+    fiducial : tuple
+        A tuple containing the world coordinates of the fiducial point.
+
+    crpix : list or tuple, optional
```

Comment:
What would be a good name for this keyword that will distinguish it from FITS? 
pixel_fiducial, pixel_reference?
Perhaps `pixel_fiducial` and `sky_fiducial`?

Reply:
I would have preferred that `crpix` be `None` by default and take the middle of the footprint instead of one of the corners.

Reply:
> What would be a good name for this keyword that will distinguish it from FITS? pixel_fiducial, pixel_reference? Perhaps `pixel_fiducial` and `sky_fiducial`?

`crpix0`